.github/workflows/verify-smart-contracts.yaml

```diff
@@ -71,6 +71,14 @@ jobs:
         with:
           version: nightly-e15e33a07c0920189fc336391f538c3dad53da73 # https://github.com/foundry-rs/foundry/releases/tag/nightly-e15e33a07c0920189fc336391f538c3dad53da73
 
+      - name: Install static analyzer
```

Comment:
```suggestion
      - name: Check for vulnarabilities
 ```

Reply:
Also, maybe we need run this step after "Link packages"? I see inside the slither script we have `yarn install`, but it not will work properly, we need run `yarn boostrap` for this, but it runned in next step.